plugins/graphiql/README.md

```diff
@@ -12,53 +12,64 @@ By exposing GraphiQL as a plugin instead of a standalone app, it's possible to p
 Start out by installing the plugin in your Backstage app:
 
 ```bash
+cd packages/app
 yarn add @backstage/plugin-graphiql
 ```
 
-Then add an entry to your App's `plugins.ts` to import the plugin.
-
-The plugin registers a `/graphiql` route, which you can link to from the Sidebar if desired.
+```diff
+# in packages/app/src/plugins.ts
++export { plugin as GraphiQL } from '@backstage/plugin-graphiql';
+```
 
-### Adding GraphQL endpoints
+```diff
```

Comment:
I have never used (seen) this definition before, and now I believe I will use it a dozen times this month 🤩 🎉 thx for that

Reply:
It's kinda neat isn't it!